sdk/include/reactos/wine/wine_common_ver.rc

```diff
@@ -51,4 +51,6 @@
 
 #define REACTOS_VERSION_DLL
 
+#define REACTOS_PRODUCTVERSION REACTOS_FILEVERSION
```

Comment:
Related:
`REACTOS_PRODUCT_VERSION` doesn't exist: typo?
```
sdk/include/reactos/wine/wine_common_ver.rc
  32 #define REACTOS_PRODUCT_VERSION WINE_PRODUCTVERSION
```
which should probably be used for
https://git.reactos.org/?p=reactos.git&a=search&h=HEAD&st=grep&s=WINE_PRODUCTVERSION